scrapy/extensions/throttle.py

```diff
@@ -15,6 +18,12 @@ def __init__(self, crawler):
 
         self.debug = crawler.settings.getbool("AUTOTHROTTLE_DEBUG")
         self.target_concurrency = crawler.settings.getfloat("AUTOTHROTTLE_TARGET_CONCURRENCY")
+        self.enable_429 = crawler.settings.getbool("AUTOTHROTTLE_HANDLE_API_RATE_LIMIT")
+        self.rate_limit_handler = None
+        self.rate_limit_status = crawler.settings.getint("AUTOTHROTTLE_RATE_LIMIT_STATUS")
+        if self.enable_429:
+            self.rate_limit_handler = AutoThrottle429Handler(
+                self.target_concurrency)
```

Comment:
:lipstick: Given the length of lines above and below, wouldn’t this be better?
```suggestion
            self.rate_limit_handler = AutoThrottle429Handler(self.target_concurrency)
```